temporal/api/cloud/identity/v1/message.proto

```diff
@@ -69,3 +69,27 @@ message User {
     // Will not be set if the user has never been modified.
     google.protobuf.Timestamp last_modified_time = 8;
 }
+
+message GroupSpec {
+    // The human-readable name of the group
+    string name = 1;
+    // The type of the group. e.g. "googleworkspace"
+    string type = 2;
+    // The access assigned to the group
+    temporal.api.cloud.identity.v1.Access access = 3;
+}
+
+message Group {
```

Comment:
I don't see a `status` field. The one that changes to `updating` when the group is being updated?

Reply:
added state and async_operation_id